### PR TITLE
📊  antibiotics: World Organisation for Animal Health

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

File: etl/steps/data/grapher/antibiotics/2024-10-23/animuse_country_level.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("animuse_country_level")
+
+    # Read table from garden dataset.
+    tb = ds_garden["animuse_country_level"]
+
+    #
+    # Process data.
```
Comment (veronikasamborska1994) [inline etl/steps/data/grapher/antibiotics/2024-10-23/animuse_country_level.py:20]: nit - can probably remove this comment

File: etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+dataset:
+  update_period_days: 365
+
+
+tables:
+  animuse_trends:
+    variables:
+      kilograms:
+        title: Quantity of antimicrobials used in animals
+        description_short: Reported quantity of antimicrobials used in animals.
```
Comment (veronikasamborska1994) [inline etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml:21]: maybe a DoD would be useful and also you should probably either use _antibiotics_ (if that's the case) or _antimicrobials_ for all of your charts? Unless of course I am missing something and you've imported antibiotics for some of these datasets and antimicrobials for others

File: etl/steps/data/garden/antibiotics/2024-10-23/animuse_year.meta.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  animuse_year:
+    variables:
+      mg_per_kg_non_adjusted:
+        title: Antimicrobial quantities adjusted for animal biomass (non-adjusted for reporting coverage)
```
Comment (veronikasamborska1994) [inline etl/steps/data/garden/antibiotics/2024-10-23/animuse_year.meta.yml:20]: you could improve the metadata here too based on some of my previous suggestions :)

File: etl/steps/data/garden/antibiotics/2024-10-23/animuse_country_level.meta.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      mg_kg__non_adjusted_by_reported_coverage:
+        title: Antimicrobial quantities adjusted for animal biomass (non-adjusted for reporting coverage)
+        description_short: Milligrams of antimicrobials used in animals per kilogram of biomass
+        unit: milligrams per kilogram
+        short_unit: mg/kg
+      mg_kg__adjusted_by_reported_coverage:
+        title: Antimicrobial quantities adjusted for animal biomass (adjusted for reporting coverage)
+        description_short: Milligrams of antimicrobials used in animals per kilogram of biomass (adjusted for reporting coverage)
+        unit: milligrams per kilogram
+        short_unit: mg/kg
```
Comment (veronikasamborska1994) [inline etl/steps/data/garden/antibiotics/2024-10-23/animuse_country_level.meta.yml:28]: I'd maybe limit the number of decimal places here and for others to 1 or even 0 for this one?

File: etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  animuse_trends:
+    variables:
+      kilograms:
+        title: Quantity of antimicrobials used in animals
```
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml:20]: Good point, I'll take a look into this and make it clearer!

File: etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml (hunk 2 above)
Comment (spoonerf) [reply etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml:21]: Ah yes, it's all antimicrobials, but the dataset is largely made up of antibiotics. I think some countries don't report exactly what they've used, and some data is confidential, so I'll stick with using antimicrobials as I think that's the most accurate option here.

**Pull request merged**